contrib/airsonic.service

```diff
@@ -24,8 +24,9 @@ Group=airsonic
 # See https://www.freedesktop.org/software/systemd/man/systemd.exec.html
 # for details
 DevicePolicy=closed
+DeviceAllow=char-alsa rw
 NoNewPrivileges=yes
-PrivateDevices=yes
+#PrivateDevices=yes
```

Comment:
If we were to remove this, should we directly remove it, rather than commenting-it-out?

Reply:
Maybe add a comment stating that this is required for Jukebox only, similar to line 46-48: #ProtectHome=true

Reply:
Yes a comment would be good and moved into a separate section like `ProtectHome`

Reply:
@Whisprin can you update your PR?

Reply:
@muff1nman Updated the PR accordingly, sorry for the delay